megazords/ios-rust/Cargo.toml

```diff
@@ -14,11 +14,8 @@ viaduct = { path = "../../components/viaduct" }
 viaduct-reqwest = { path = "../../components/support/viaduct-reqwest" }
 nimbus-sdk = { path = "../../components/nimbus" }
 crashtest = { path = "../../components/crashtest" }
-
-# TODO: can't include fxa-client until we get NSS working on M1 simulator,
-# ref https://github.com/mozilla/application-services/issues/4352.
-#fxa-client = { path = "../../components/fxa-client" }
-# TODO: can't include logins until we get SQLCipher working on M1 simulator,
-# ref https://github.com/mozilla/application-services/issues/4352.
-# (or until we entirely get rid of SQLCipher)
-#logins = { path = "../../components/logins" }
+fxa-client = { path = "../../components/fxa-client" }
+logins = { path = "../../components/logins" }
+autofill = { path = "../../components/autofill" }
+push = { path = "../../components/push" }
```

Comment:
I'm curious what should be our approach for components that aren't being consumed in iOS yet **at all**, like `autofill` and `push` if we add them here, then we force the scripts to compile the rust code and we generate the headers for them.. not that it's a problem, the scripts shouldn't run often (unless local dev will be impacted?)

The alternative would be to omit them, and add them when/if we get an iOS consumer that wants them

Thoughts?

Reply:
Yeah I went back and forth on this a little trying to decide the best way but the pros kinda outweigh the cons at this point:

Pros - component completeness, ensuring all components can build with the new system, able to be consumed with no need of a PR/Cut/release, since they select the components on consumption -- no extra overhead

Cons - little longer CI build times (which we'll eventually change to only generate artifacts on release builds)

If for some reason we start seeing really long build times, issues, I think it's okay to remove until ready for consumption.

Reply:
I'm convinced 💯